src/main/java/it/pagopa/ecommerce/commons/domain/v2/pojos/BaseTransactionWithRefundRequested.java

```diff
@@ -20,19 +24,45 @@
 public abstract class BaseTransactionWithRefundRequested extends BaseTransactionWithRequestedAuthorization {
 
     BaseTransactionWithRequestedAuthorization transactionAtPreviousState;
+    @Nullable
+    TransactionGatewayAuthorizationData refundRequestedAuthorizationGatewayData;
 
     /**
      * Primary constructor
      *
-     * @param baseTransaction base transaction
+     * @param baseTransaction                         base transaction
+     * @param refundRequestedAuthorizationGatewayData refund requested optional
+     *                                                authorization gateway data
```

Comment:
I would add a few words here too explaining when it can be null :)

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/199/commits/61b275cd5faf02a3af53c41af45ac873b74961aa